spec/intelligent_foods/resources/menu_spec.rb

```diff
@@ -24,4 +24,79 @@
       end
     end
   end
+
+  describe ".find" do
+    it "returns the menu" do
+      menu_id = "2023-01-01"
+      body = build_menu_response(menu_id: menu_id)
+      response = build_response(body: body)
+      stub_api_response response: response
+
+      result = IntelligentFoods::Menu.find(menu_id)
```

Comment:
when working within the same module you dont need to use the namespace

Reply:
@jmazur Very possible that I am misunderstanding something, but deleting that namespace breaks the tests. Possibly not doing something right.